src/Entity/Organization/Package.php

```diff
@@ -310,4 +310,14 @@ public function keepLastReleases(): int
     {
         return $this->keepLastReleases;
     }
+
+    public function setKeepLastReleases(int $keepLastReleases): void
+    {
+        $this->keepLastReleases = $keepLastReleases;
+    }
+
+    public function setRepositoryUrl(string $url): void
```

Comment:
If we always edit this values together, then this should be one logical domain method (editPackage or update)